chroma-manager/tests/framework/utils/defaults.sh

```diff
@@ -96,4 +96,23 @@ set_defaults() {
     set_distro_vars "$JENKINS_DISTRO"
 
     export CLUSTER_CONFIG="cluster_cfg.json"
+
+    export COPR_OWNER="managerforlustre"
+    export COPR_PROJECT="manager-for-lustre"
+    # if you use this, don't forget to update chroma-manager/chroma_agent_comms/views.py
```

Comment:
This sounds kind of brittle. Is there anything we can do so that we don't have to think about this step?

Reply:
Yeah.  I'm not entirely a fan of it myself.  The complication is that it spans both the production application as well as our test framework.  Nothing sprang to mind about how to centralise it into one place that both could use easily.

I'm open to ideas.

Reply:
Fair enough. This is probably something we may want to discuss in the future but I don't see it as a blocker. I'll approve.

Reply:
Agreed on figuring out a more elegant, single-point-of-data solution at some point.  I have some possible ideas, just not any time really to see what works and what doesn't.